core/src/main/java/com/scottlogic/datahelix/generator/core/generation/string/generators/FakerGenerator.java

```diff
@@ -71,6 +72,24 @@ public boolean validate(String string) {
     }
 
     private String getFakerValue(Faker faker) {
-        return faker.expression("#{" + this.fakerSpec + "}");
+        try {
+            return faker.expression("#{" + this.fakerSpec + "}");
+        } catch (Exception e) {
+            return getFakerValueWorkaround(faker);
+        }
+    }
+
+    private String getFakerValueWorkaround(Faker faker) {
```

Comment:
Do you think it would be worth adding a comment with an explaination about why this tempory workaround is required and perhaps a link to the java faker issue that needs to be resolved?

Reply:
Good idea, I meant to do this and forgot :)